instrumentation-docs/src/test/java/io/opentelemetry/instrumentation/docs/utils/YamlHelperTest.java

```diff
@@ -61,21 +63,27 @@ void testPrintInstrumentationList() throws Exception {
     writer.flush();
 
     String expectedYaml =
-        "spring:\n"
-            + "  instrumentations:\n"
-            + "  - name: spring-web-6.0\n"
-            + "    description: Spring Web 6.0 instrumentation\n"
-            + "    srcPath: instrumentation/spring/spring-web/spring-web-6.0\n"
-            + "    target_versions:\n"
-            + "      javaagent:\n"
-            + "      - org.springframework:spring-web:[6.0.0,)\n"
-            + "struts:\n"
-            + "  instrumentations:\n"
-            + "  - name: struts-2.3\n"
-            + "    srcPath: instrumentation/struts/struts-2.3\n"
-            + "    target_versions:\n"
-            + "      library:\n"
-            + "      - org.apache.struts:struts2-core:2.1.0\n";
+        """
```

Comment:
nice idea with the multiline strings, no reason this module can't target latest JDK 👍